src/audio/template_comp/template_comp.toml

```diff
@@ -0,0 +1,21 @@
+#ifndef LOAD_TYPE
```

Comment:
I would also put a lot cf comments in here too.

Reply:
Unfortunately I don't know the meaning of these. We have a document start in https://github.com/thesofproject/sof-docs/pull/451/files but it was never merged to sof-docs where it should be described.